lib/compiler/src/compile.erl

```diff
@@ -719,7 +719,9 @@ core_passes() ->
 
 kernel_passes() ->
     %% Optimizations that must be done after all other optimizations.
-    [{pass,sys_core_bsm},
+    [{pass,sys_core_alias},
```

Comment:
I simply placed it close to the `sys_core_bsm` and `sys_core_dsetel` passes. I do not have enough context to decide where is the best place to the pass, so I will gladly defer to your judgement. :)